airflow/www/templates/airflow/dag.html

```diff
@@ -33,7 +33,8 @@ <h3 class="pull-left">
         <span style='color:#AAA;'>SUBDAG: </span> <span> {{ dag.dag_id }}</span>
       {% else %}
         <input id="pause_resume" dag_id="{{ dag.dag_id }}" type="checkbox" {{ "checked" if not dag.is_paused else "" }} data-toggle="toggle" data-size="mini" method="post">
-        <span style='color:#AAA;'>DAG: </span> <span> {{ dag.dag_id }}</span> <small class="text-muted"> {{ dag.description }} </small>
+        <span style='color:#AAA;'>DAG: </span> <span> {{ dag.dag_id }}</span>
+        <small class="text-muted"> description: {{ dag.description[0:150] + '...' if dag.description and dag.description|length > 150 else dag.description }} </small>
```

Comment:
Shouldn't we hide this text if dag.description == None?

Reply:
I want to keep same UI behavior and view, so I did it. Do you think we should hide text

Reply:
I think it's better to hide it. Some users do not use descriptions, so the repetitive text "None" can be annoying.